src/messages.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ * Wraps the content inside a codeblock with no language.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function codeBlock<C extends string>(content: C): `\`\`\`\n${C}\`\`\``;
+/**
+ * Wraps the content inside a codeblock with the specified language.
+ * @param language The language for the codeblock.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function codeBlock<L extends string, C extends string>(language: L, content: C): `\`\`\`${L}\n${C}\`\`\``;
+export function codeBlock(language: string, content?: string): string {
+  return typeof content === 'undefined' ? `\`\`\`\n${language}\`\`\`` : `\`\`\`${language}\n${content}\`\`\``;
+}
+
+/**
+ * Wraps the content inside an inline code.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function inlineCode<C extends string>(content: C): `\`${C}\`` {
+  return `\`${content}\``;
+}
+
+/**
+ * Formats the content into italic text.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function italic<C extends string>(content: C): `*${C}*` {
+  return `*${content}*`;
+}
+
+/**
+ * Formats the content into bold text.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function bold<C extends string>(content: C): `**${C}**` {
+  return `**${content}**`;
+}
+
+/**
+ * Formats the content into underscored text.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function underscore<C extends string>(content: C): `__${C}__` {
+  return `__${content}__`;
+}
+
+/**
+ * Formats the content into strikethrough text.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function strikethrough<C extends string>(content: C): `~~${C}~~` {
+  return `~~${content}~~`;
+}
+
+/**
+ * Formats the content into a quote. This needs to be at the start of the line for Discord to format it.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function quote<C extends string>(content: C): `> ${C}` {
+  return `> ${content}`;
+}
+
+/**
+ * Formats the content into a block quote. This needs to be at the start of the line for Discord to format it.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function blockQuote<C extends string>(content: C): `>>> ${C}` {
+  return `>>> ${content}`;
```

Comment:
Is this one still valid nowadays? Referring to bots, not clients 👀

Reply:
Replied in Discord.